app/models/meeting.rb

```diff
@@ -56,7 +77,15 @@ def set_slug
     end
   end
 
-  def attendees_array
-    invitations.accepted.map { |a| [a.member.full_name] }
+  def set_date_and_time
+    new_date_and_time = datetime_from_fields(local_date, local_time)
+    self.date_and_time = new_date_and_time if new_date_and_time
+  end
+
+  def datetime_from_fields(date_string, time_string)
```

Comment:
Since `set_date_and_time` and `datetime_from_fields` are reused across `Meeting` and `Workshop` can we extract them in a reusable module to remove duplication. `eg DateTimeConcerns`